src/__internal__/character-count/character-count.spec.tsx

```diff
@@ -17,13 +21,75 @@ describe("CharacterCount", () => {
     it("should render default", () => {
       assertStyleMatch(
         {
-          textAlign: "right",
+          textAlign: "left",
           fontSize: "12px",
           marginTop: "4px",
           marginBottom: "4px",
           color: "var(--colorsUtilityYin055)",
         },
-        wrapper
+        wrapper.find(StyledCharacterCount)
+      );
+    });
+
+    it("character counter has aria-hidden attribute", () => {
+      expect(wrapper.find(StyledCharacterCount).prop("aria-hidden")).toBe(
+        "true"
+      );
+    });
+
+    it("character counter has correct count", () => {
+      expect(wrapper.find(StyledCharacterCount).text()).toBe(
+        "5 characters left"
+      );
+    });
+
+    it("visually hidden character counter has aria-live='polite'", () => {
+      expect(wrapper.find(VisuallyHiddenCharacterCount).prop("aria-live")).toBe(
+        "polite"
+      );
+    });
+
+    it("visually hidden character counter has the same count message as visual character counter", () => {
+      expect(wrapper.find(VisuallyHiddenCharacterCount).text()).toBe(
+        "5 characters left"
+      );
+    });
+
+    it("visually hidden character counter has css properties to visually hide it from view", () => {
+      assertStyleMatch(
+        {
+          border: "0",
+          height: "1px",
+          margin: "-1px",
+          overflow: "hidden",
+          padding: "0",
+          position: "absolute",
+          width: "1px",
+          whiteSpace: "nowrap",
+        },
+        wrapper.find(VisuallyHiddenCharacterCount)
+      );
+    });
+
+    it("visually hidden hint has correct message based on 'characterLimit", () => {
+      expect(wrapper.find(VisuallyHiddenHint).text()).toBe(
+        "You can enter up to 10 characters"
+      );
+    });
+
+    it("visually hidden hint has visuallyHidden class applied", () => {
```

Comment:
**suggestion**: we should update this test in a similar fashion to the one on line 52.

```suggestion
    it("visually hidden hint has css properties to visually hide it from view", () => {
```